_posts/2021-06-07-How-to-edit-names-if-you-have-made-spelling-mistakes-in-the-lead-name-itself.md

```diff
@@ -0,0 +1,59 @@
+---
+date: 2021-06-07
+title: How to edit names of lead if you have made spelling mistakes in the lead name itself
```

Comment:
Need to change the title to below:
How to edit lead name if you have made spelling mistake